src/components/home/modals/CreateToken.vue

```diff
@@ -711,6 +711,17 @@ export default {
                 sendTx = this.dataObject.toJsonForSendingTx(signature)
             }
             if (this.selectedNFTContract) {
+                let firstTokenId = common.contractIDToTokenID(this.nftContractID)
+                let firstTokenIndex = common.getTokenIndex(firstTokenId)
+                let newTokenIndex = firstTokenIndex
```

Comment:
This convert is useless. Because `firstTokenIndex` is always 0.

Reply:
The best solution is find the max index that contract ID are the same in Token Watch List and then start to loop. If no, start index from 0.